pkg/config/signer.go

```diff
@@ -149,21 +146,17 @@ func (s *Signer) CreateSignatureHeader() (*common.SignatureHeader, error) {
 
 // createNonce generates a nonce using the crypto/rand package.
 func createNonce() ([]byte, error) {
-	nonce, err := getRandomNonce()
+	nonce := make([]byte, 24)
+
+	_, err := rand.Read(nonce)
 	if err != nil {
-		return nil, fmt.Errorf("failed to generate random nonce: %s", err)
+		return nil, fmt.Errorf("failed to get random bytes: %s", err)
 	}
-	return nonce, nil
-}
-
-func getRandomNonce() ([]byte, error) {
-	key := make([]byte, 24)
 
-	_, err := rand.Read(key)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get random bytes: %s", err)
+		return nil, fmt.Errorf("failed to generate random nonce: %s", err)
 	}
-	return key, nil
+	return nonce, nil
 }
 
 // When using ECDSA, both (r,s) and (r, -s mod n) are valid signatures.
```

Comment:
nit: This function isn't exported so it doesn't matter that much but exported function comments for GoDoc generally need to start with the function name. I would try to reorganize this function comment to start with `toLowS ...`

Reply:
done